create-swarm-secret.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Can we convert the code to run with `sh` instead?

Reply:
no idea, why would you want to?

Reply:
Not all platforms have bash available by default I.e. Alpine Linux.